src/core/types/message.rs

```diff
@@ -4,17 +4,196 @@ use crate::core::types::{
     SuccessfulPayment, User, Venue, Video, VideoNote, Voice,
 };
 
-#[derive(Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[derive(Debug, Deserialize, Eq, Hash, PartialEq)]
 pub struct Message {
     #[serde(rename = "message_id")]
-    pub id: i32,
+    pub id: i64,
     pub date: i32,
-    pub chat: Chat,
+    pub chat: Box<Chat>,
```

Comment:
Why there is `box`? In `Chat` there is only `Box<Message>`, but not `Message`...

Reply:
I thought it would be nice to make relationship symmetrical

Reply:
I think that one of these boxes is redundant. Also, I think that `Message.chat` is used more often than `Chat.pinned_message` so we should remove boxing here.

Reply:
Okay